js/functions.js

```diff
@@ -0,0 +1,39 @@
+/*Функция для проверки длины строки. Она принимает строку, которую нужно проверить, и максимальную длину и возвращает true,
+если строка меньше или равна указанной длине, и false, если строка длиннее. Эта функция
+нам пригодится для валидации формы. Примеры использования функции: */
+
+
+const compareStroke = (line, symbol) => line.length <= symbol.length;
+
+compareStroke();
+
+/*Функция для проверки, является ли строка палиндромом. Палиндром — это слово или фраза, которые одинаково читаются и слева направо и справа налево.*/
+const isPalindrom = (stroke) => {
+  stroke = stroke.replaceAll(' ','');
+  stroke = stroke.toLowerCase();
+  let result = '';
+  for (let i = stroke.length - 1; i >= 0 ; i--) {
+    result += stroke[i];
+  }
+  return stroke === result;
+};
```

Comment:
циклы и return лучше всегда отделять пустой строкой
```suggestion
  let result = '';
  
  for (let i = stroke.length - 1; i >= 0 ; i--) {
    result += stroke[i];
  }
  
  return stroke === result;
};
```